src/main/java/org/fogbowcloud/saps/engine/core/archiver/Archiver.java

```diff
@@ -181,51 +99,49 @@ private void garbageCollector() {
 
         for (SapsImage task : failedTasks)
             deleteAllTaskFilesFromDisk(task);
-
     }
 
     /**
-     * This function gets tasks in failed state in Catalog.
+     * It gets tasks in failed state in {@code Catalog}.
      *
-     * @return tasks in specific state
+     * @return {@code SapsImage} list in {@code ImageTaskState.FAILED} state
```

Comment:
does it return an empty list, when there is no failed task? doc-it

Reply:
Done.